worker/stream.go

```diff
@@ -60,6 +60,11 @@ func stream(streamCtx *StreamContext) {
 		if errFfmpegErrFile == nil {
 			_ = ffmpegErr.Close()
 		}
+		// Stream start seems to be successful, so we can save the stream
+		// This request is only used for lecture hall streams
+		if !streamCtx.isSelfStream {
+			lectureHallStreams[streamCtx.streamId] = append(lectureHallStreams[streamCtx.streamId], streamCtx)
+		}
```

Comment:
This should not be appended here, the context pointer should not change, so it's fine to add the context to the slice in the request handler.

Reply:
Okay, I will move it :+1: